loris/resolver.py

```diff
@@ -177,20 +178,10 @@ def is_resolvable(self, ident):
         else:
             fp = self._web_request_url(ident)
 
-            if self.head_resolvable:
```

Comment:
Looking at it now, I'm not sure why.  My best guess is that I got tangled up in the circular symlinks (https://github.com/loris-imageserver/loris/issues) and this was a way to short-circuit while testing.  Whatever the case, and my apologies for letting that persist into the pull request, I think you're right that it should stay.

If you want to go with the `with closing(...` syntax, that'd mean rewriting the requests from lines 181 - 193 of the original code to use that syntax.

What's your preference for doing this?  Would it be easier if I created a new pull request?